README.md

```diff
@@ -55,7 +55,7 @@ npm start
 | Command | Description |
 | ------- | ----------- |
 | `npm start` | Starts development server with hot reloading and api proxy. |
-| `npm test` | Runs all the test |
+| `npm test -- --coverage` | Runs all the tests |
```

Comment:
shouldn't we have `npm test` work without arguments?

Reply:
For better or worse, if it's a git repo jest and react-scripts will "detect" tests that need to run (fairly successfully) based on the current change set. Net result is, when there aren't any changes, you're presented with options:

```
No tests found related to files changed since last commit.
Press `a` to run all tests, or run Jest with `--watchAll`.

Watch Usage
 › Press a to run all tests.
 › Press p to filter by a filename regex pattern.
 › Press t to filter by a test name regex pattern.
 › Press q to quit watch mode.
 › Press Enter to trigger a test run.
```

Found a hack to avert this and added a script `test-dev` that retains the old behavior (which is convenient for development).